guides/v2.3/graphql/reference/store-config.md

```diff
@@ -1,38 +1,80 @@
 `base_link_url` | String | A fully-qualified URL that is used to create relative links to the `base_url`
 `base_static_url` | String | The fully-qualified URL that specifies the location of static view files, such as `http://magentohost.example.com/pub/static/`
 `base_media_url` | String | The fully-qualified URL that specifies the location of user media files, such as `http://magentohost.example.com/pub/media/`
-`secure_base_url` | String | The store's fully-qualified secure base URL, such as `https://magentohost.example.com/`
-`secure_base_link_url` | String | A fully-qualified URL that is used to create relative links to the `base_url`
-`secure_base_static_url` | String | The secure fully-qualified URL that specifies the location of static view files, such as `https://magentohost.example.com/pub/static/`
+`base_url` | String | The store's fully-qualified base URL, such as `http://magentohost.example.com/`
+`code` | String | A unique identifier for the store
+`default_display_currency_code` | String | The code representing the currency displayed on the store, such as `USD`
+`id` | Integer | The ID number assigned to the store
+`locale` | String | The store's locale, such as `en_US`
+`secure_base_link_url` | String | A secure fully-qualified URL that is used to create relative links to the `base_url`
 `secure_base_media_url` | String | The secure fully-qualified URL that specifies the location of user media files, such as `https://magentohost.example.com/pub/media/`
+`secure_base_static_url` | String | The secure fully-qualified URL that specifies the location of static view files, such as `https://magentohost.example.com/pub/static/`
+`secure_base_url` | String | The store's fully-qualified secure base URL, such as `https://magentohost.example.com/`
+`timezone` | String | The store's time zone, such as `America/Chicago`
+`website_id` | Integer | The ID number assigned to the parent website  
+`weight_unit` | String | The weight unit for products, such as `lbs` or `kgs`
+{:style="table-layout:auto;"}
+
+### Supported theme attributes
+
+Use the `theme` attributes to retrieve information about the store's thematic elements; such as, footer and header information, copyright text, and logo information.
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`absolute_footer` | String | Contains scripts that must be included in the HTML before the closing &lt;body&gt; tag
+`copyright` | String | The copyright statement that appears at the bottom of each page
+`default_description` | String | The description that provides a summary of your site for search engine listings and should not be more than 160 characters in length
+`default_keywords` | String | A series of keywords that describe your store, each separated by a comma
+`default_title` | String | The title that appears at the title bar of each page when viewed in a browser
+`demonotice` | Int | Controls the display of the demo store notice at the top of the page. Options include: Yes / No
```

Comment:
The value can't be Yes/No when the value is an Int

Reply:
fixed